rapid7_insightvm/komand_rapid7_insightvm/actions/asset_vulnerability_solution/action.py

```diff
@@ -0,0 +1,39 @@
+import komand
+from .schema import AssetVulnerabilitySolutionInput, AssetVulnerabilitySolutionOutput, Input, Output, Component
+# Custom imports below
+import asyncio
+from komand_rapid7_insightvm.util import endpoints
+
+
+class AssetVulnerabilitySolution(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='asset_vulnerability_solution',
+                description=Component.DESCRIPTION,
+                input=AssetVulnerabilitySolutionInput(),
+                output=AssetVulnerabilitySolutionOutput())
+
+    def run(self, params={}):
+        asset_id = params.get(Input.ASSET_ID)
+        vulnerability_ids = params.get(Input.VULNERABILITY_IDS)
+
+        solutions = asyncio.run(self.async_get_asset_vulnerability_solutions(asset_id, vulnerability_ids))
+        return {Output.VULNERABILITIES_SOLUTION: solutions}
+
+    async def async_get_asset_vulnerability_solutions(self, asset_id: str, vulnerability_ids: [str]) -> [dict]:
```

Comment:
Is there any intention to restrict the number of vulnerability IDs to provide here?  Are we indifferent if the customer gets a source of vulnerability IDs in the tens of thousands?